### PR TITLE
feat: static page generator for hackathons

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

With this PR -> It is possible to build out a static page under 10-15min

GUIDELINE: It **MUST** follow the same structure as shown [here](https://user-images.githubusercontent.com/5358146/59133692-6c80f080-8996-11e9-9450-0bf60835c777.png) which was built out by @willsputra 

Any time a hackathon page is built out, do ensure 

- color scheme (color hex code)
- text as per the layout 
- all images - hackathon logo + sponsors ( same size ) 
- all outgoing links are provided 
- the better the image resolution -> the neater the site will look

I've redone the ethhack page by simply updating [this file](https://github.com/thelostone-mc/web/blob/static/app/dashboard/context/hackathon.py).

Creating a new hackthon is as simple as making a new object just like that with updated info (5-10min) & updating the color scheme (5 min) and that's about it 

**DEMO** : [SEE HERE](https://embed.vidyard.com/share/GSGB8VZA1Y5q5Uxh7i1Mc5?)

##### Tech Description 

- code refactor
- removed old eth hack static page
- moved context into it's own folder to keep views cleaner
- restructured files to group hackathon + pages + assets

fixes https://github.com/gitcoinco/web/issues/4508

### CODE REVIEW

**Comment by frankchen07** [comment]
dope - with music again too. the best

**Comment by thelostone-mc** [comment]
@danlipert 

> How do you set the URL for each hackathon static page? 
Also, will this support having multiple hackathon static pages up at the same time?

Yes and Yes
You'd simply define a new url route and define a view which has a custom config.
The all use the same template hackathon 🙌

**Comment by danlipert** [comment]
> @danlipert
> 
> > How do you set the URL for each hackathon static page?
> > Also, will this support having multiple hackathon static pages up at the same time?
> 
> Yes and Yes
> You'd simply define a new url route and define a view which has a custom config.
> The all use the same template hackathon 🙌

niiice - would be dope if we could set those up via the context or another config file - even less work for us ;) but this is a great start!

**Comment by thelostone-mc** [comment]
@danlipert you mean like [this](https://github.com/gitcoinco/web/pull/4594/files#diff-6681798972c2cd3cdc546fec9a3975ee) ? :P the view is cleannnnn ✨

File: app/assets/v2/images/hackathon/ethhack/banner.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+<svg width="3009px" height="609px" viewBox="0 0 3009 609" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g id="Page-2" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <rect id="Rectangle" fill="#180944" fill-rule="nonzero" x="0" y="0" width="3009" height="609"></rect>
+        <g id="image-1440x480" transform="translate(1346.000000, 140.000000)">
```
Comment (octavioamu) [inline app/assets/v2/images/hackathon/ethhack/banner.svg]: we have this image without being base64 here https://github.com/gitcoinco/web/blob/master/app/assets/v2/images/hackathon/ethereal.svg#L1

**Pull request merged**